src/mat_tables.c

```diff
@@ -93,13 +95,10 @@ static const RULE rules[] = {
   { "0  0  0      0      0  1  0  2  0      0      0  0", {     0,   DRAWN} }, /* Q vs N+N */
   { "0  2  0      0      0  0  0  0  0      0      0  1", {     0,   DRAWN} },
 
-  { "0  1  0      0      0  1  0  0  0      0      0  1", {     0,   DRAWN} }, /* Q+N vs Q */
-  { "0  0  0      0      0  1  0  1  0      0      0  1", {     0,   DRAWN} },
-
-  { "0  0  1      0      0  1  0  0  0      0      0  1", {     0,   DRAWN} }, /* Q+B vs Q */
-  { "0  0  0      1      0  1  0  0  0      0      0  1", {     0,   DRAWN} },
-  { "0  0  0      0      0  1  0  0  1      0      0  1", {     0,   DRAWN} },
-  { "0  0  0      0      0  1  0  0  0      1      0  1", {     0,   DRAWN} },
+  { "a+b+c=1                                           ", {     0,   CONSTRAINT   } },
+  { "0  a  b      c      0  1  0  0  0      0      0  1", {     0,   DRAWN} }, /* Q+B vs Q */
+  { "0  0  0      0      0  1  0  a  b      c      0  1", {     0,   DRAWN} },
+  { NULL,                                                 {     0,   CONSTRAINT   } },
 
   { ">0 0  0      0      0  0  0  1  0      0      0  0", {      KNIGHT_V,   0} }, /* piece can't win, against pawn, */
```

Comment:
so this https://lichess.org/YnsBunen/black#81 is why this wan't such a good idea. Taking away the full bishop value makes the position equal to positions where they don't have the bishop